lib/assessment.rb

```diff
@@ -36,4 +57,13 @@
 
 ##
 # your code here
+class Developer
+  attr_accessor :hire_for_job
+  def initialize(hire_for_job)
+    @hire_for_job = hire_for_job
```

Comment:
This is not a `hire_for_job` method. What is `hire_for_job` on this line of code?